pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/reporter/BrokerLoadDataReporter.java

```diff
@@ -77,12 +79,18 @@ public BrokerLoadData generateLoadData() {
         final var pulsarStats = pulsar.getBrokerService().getPulsarStats();
         synchronized (pulsarStats) {
             var brokerStats = pulsarStats.getBrokerStats();
+            var bundleStats = pulsarStats.getBundleStats();
+            var topics = 0;
+            for (Map.Entry<String, NamespaceBundleStats> bundleStatsEntry : bundleStats.entrySet()) {
```

Comment:
Can we define `topics` in `BrokerStats` and pre-aggregate this topic counts in `updateStats`?
```

   public synchronized void updateStats(
            ConcurrentOpenHashMap<String, ConcurrentOpenHashMap<String, ConcurrentOpenHashMap<String, Topic>>>
                    topicsMap) {
...
                    bundles.forEach((bundle, topics) -> {
                        NamespaceBundleStats currentBundleStats = bundleStats.computeIfAbsent(bundle,
                                k -> new NamespaceBundleStats());
                        currentBundleStats.reset();
                        currentBundleStats.topics = topics.size();
                        brokerStats.topics += topics.size(); //pre-aggregate

```

Reply:
Yes, pre-aggregate will be better. Updated.